apps/dokploy/components/dashboard/settings/servers/gpu-support.tsx

```diff
@@ -0,0 +1,262 @@
+import { AlertBlock } from "@/components/shared/alert-block";
+import { DialogAction } from "@/components/shared/dialog-action";
+import { Button } from "@/components/ui/button";
+import {
+	Card,
+	CardContent,
+	CardDescription,
+	CardHeader,
+	CardTitle,
+} from "@/components/ui/card";
+import { api } from "@/utils/api";
+import { TRPCClientError } from "@trpc/client";
+import { CheckCircle2, Cpu, Loader2, XCircle } from "lucide-react";
+import { useState } from "react";
+import { toast } from "sonner";
+
+interface GPUSupportProps {
+	serverId?: string;
+}
+
+export function GPUSupport({ serverId }: GPUSupportProps) {
+	const [isLoading, setIsLoading] = useState(false);
+	const utils = api.useContext();
+
+	const { data: gpuStatus, isLoading: isChecking } =
+		api.settings.checkGPUStatus.useQuery(
+			{ serverId },
+			{
+				enabled: serverId !== undefined,
+				refetchInterval: 5000,
```

Comment:
Can you remove this?

I think is better to have a refresh button instead of fetching the data in this interval